tests/components/file_info_preview.test.jsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+
+import {shallow} from 'enzyme';
+
+import FileInfoPreview from 'components/file_info_preview.jsx';
+
+describe('components/file_info_preview.jsx', () => {
+    const requiredProps = {
+        fileUrl: 'http://www.catphotos.org/wp-content/uploads/2013/02/cat-standing.jpg',
```

Comment:
Could you just change this to `https://pre-release.mattermost.com/api/v4/files/rqir81f7a7ft8m6j6ej7g1txuo`?

Reply:
Can do! 
Sorry I couldn't think of a more suitable link to test with

Reply:
No problem.
Actually, using fake URL is also desirable as we are not really rendering the actual component. But in case of using real URL then, related to Mattermost is preferable.